packages/components/input/src/InputAddon.tsx

```diff
@@ -5,19 +5,13 @@ import { useInputGroup } from './InputGroupContext'
 
 export interface InputAddonProps
   extends ComponentPropsWithoutRef<'div'>,
-    Omit<InputAddonStylesProps, 'intent' | 'isDisabled' | 'isHovered' | 'isFocused'> {}
+    Omit<InputAddonStylesProps, 'intent' | 'isDisabled'> {}
 
 export const InputAddon = forwardRef<HTMLDivElement, PropsWithChildren<InputAddonProps>>(
   ({ className, ...others }, ref) => {
-    const { intent, isDisabled, isHovered, isFocused } = useInputGroup() || {}
+    const { isDisabled } = useInputGroup() || {}
```

Comment:
Instead of using the `||` operator every time at the call site, couldn't we handle it inside `useInputGroup` instead?, i.e.:
```tsx
export const useInputGroup = () => {
  return useContext(InputGroupContext) || {}
}
```

Reply:
Sure will change!